cocos2d/audio/CCAudio.js

```diff
@@ -591,9 +591,15 @@ cc.Audio = cc.Class.extend({
                         cb(null, audio);
                     }, function(){
                         //error
+                        cb({status:520, errorMessage:ERRSTR}, null);
```

Comment:
Oh, sorry, this is my mistake, please check whether `loader.loadAudioFromExtList` handled the callback, if yes, you can remove this cb